### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Luni-4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,10 @@
 - Install `bubblesub`: `pip install --user .`
 - Run bubblesub: `python3 -m bubblesub` or simply `bubblesub`
 
+If you want to simplify `bubblesub` installation, look at our [Dockerfile](Dockerfile)!
```
Comment (rr-) [inline README.md]:
```diff
-!
+.
```

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
     Simple extensible ASS subtitle editor for Linux
 </p>
 
-## Features
+## Features [![Travis Build Status](https://travis-ci.org/rr-/bubblesub.svg?branch=master)](https://travis-ci.org/rr-/bubblesub)
```
Comment (rr-) [inline README.md]: It's a weird place to put this, let's put it centered above this heading

File: README.md (hunk 1 above)
Comment (Luni-4) [reply README.md]: I liked the emphasis given by the exclamation mark, but done

File: README.md (hunk 2 above)
Comment (Luni-4) [reply README.md]: I prefer where it was, but done.

File: README.md [hunk 3]
```diff
@@ -4,8 +4,14 @@
 
 <p align='center'>
     Simple extensible ASS subtitle editor for Linux
+<br />
+<br />
+<a href="https://travis-ci.org/rr-/bubblesub">
+    <img src="https://travis-ci.org/rr-/bubblesub.svg?branch=master">
+</a>
```
Comment (rr-) [inline README.md]: Now the HTML is using wrong indentation.

Comment (Luni-4) [reply README.md]: Oh, sorry. Done.

**Pull request merged**